context/app/static/js/components/cells/DatasetClusterChart/DatasetClusterChart.jsx

```diff
@@ -31,7 +31,9 @@ function DatasetClusterChart({ uuid, results }) {
   });
 
   const xScale = scaleBand({
-    domain: selectedData.map((d) => d.cluster_number).sort((a, b) => a - b),
+    domain: selectedData
+      .sort((a, b) => b.unmatched + b.matched - (a.unmatched + a.matched))
```

Comment:
Slight preference to pull out a helper function?
```suggestion
      .sort((a, b) => all(b) - all(a))
```
```javascript
function all({matched, unmatched}) {
  return matched + unmatched;
}
```

Reply:
Done, thanks!